docs/UserGuide.md

```diff
@@ -81,6 +83,16 @@ e.g. `07:00` is a valid time input and refers to 7am, but not `7:00`
+|-------------|-----------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Name**    | `-name`   | Yes      | Names should only contain alphanumeric characters and spaces, it cannot contain special character, do note that long names might cause the Id field to be blocked, in this case you are able to drag the middle portion of the UI to reveal the ID, the upper limit of this workaround would be if the name is 220 characters long. In this case do proceed to use nicknames instead. |
+| **Phone**   | `-phone`  | Yes      | Phone numbers should only contain numbers, and it should be at least 3 digits long <br/> There is no limit on the length of the phone number to be accepted worldwide.                                                                                                                                                                                                                |
+| **Email**   | `-email`  | Yes      | Emails does not require top level domain, it just require an @, for example user@com                                                                                                                                                                                                                                                                                                  |
+| **Address** | `-address` | Yes      | Addresses can take any values, and it should not be blank                                                                                                                                                                                                                                                                                                                             |
+| **Subject** | `-subject` | Yes      | Subject can take any values, and it should not be blank.                                                                                                                                                                                                                                                                                                                              |
+| **Tag**     | `t/`      | Nope     | Tags names should be alphanumeric and not contain any special character.                                                                                                                                                                                                                                                                                                              | 
```

Comment:
Not sure about this but maybe special characters should be plural? 
`Tags should be alphanumeric and not contain any special characters.`